reference/orders.v2.oas2.yml

```diff
@@ -4053,6 +4054,11 @@ components:
           description: Tracking number of the shipment.
           example: w4se4b6ASFEW4T
           maxLength: 50
+        tracking_link:
```

Comment:
is this field only supported in POST ?

Reply:
It should be completely writable. I have updated it for PUT and POST now.